<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>625</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>687</width>
    <height>593</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>UVSim</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="memoryTable">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>251</width>
      <height>571</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="rowCount">
     <number>100</number>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <row>
     <property name="text">
      <string>0</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>7</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>9</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>10</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>11</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>12</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>13</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>14</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>15</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>16</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>17</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>18</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>19</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>20</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>21</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>22</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>23</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>24</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>25</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>26</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>27</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>28</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>29</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>30</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>31</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>32</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>33</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>34</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>35</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>36</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>37</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>38</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>39</string>
     </property>
    </row>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column>
     <property name="text">
      <string>Memory Value</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Operation</string>
     </property>
    </column>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>611</width>
      <height>561</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QSplitter" name="splitter_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>551</width>
       <height>291</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QTextBrowser" name="output_window"/>
    </widget>
    <widget class="QTextBrowser" name="accumulator_text">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>370</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="pc_text">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>410</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="pc_data_text">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>450</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="help_button">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>500</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="run_button">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>500</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>RUN</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>340</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Input</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Output</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>370</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Accumulator:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>410</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;PC:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>450</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Data at PC:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="input_window">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>370</y>
       <width>181</width>
       <height>171</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="input_button">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>340</y>
       <width>71</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter Input</string>
     </property>
    </widget>
    <widget class="QPushButton" name="output_clear_button">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="reset_button">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>500</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>RESET</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>890</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="menu_save"/>
    <addaction name="menu_load"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="menu_save">
   <property name="text">
    <string>Save Memory To File</string>
   </property>
  </action>
  <action name="menu_load">
   <property name="text">
    <string>Load Memory from File</string>
   </property>
  </action>
  <action name="actionThing2">
   <property name="text">
    <string>Thing2</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
